### PR TITLE
Add an option to create a new notebook

### PR DESCRIPTION
**Pull request by senderle**
This code adds a simple command line option to create a new notebook if one doesn't already exist. It does some obvious checks to see if the file exists (either with or without the `.ipynb` extension) but I may have missed something.

I saw that you said elsewhere that you see this as mainly for GUI integration, but I've been using nbopen at the console regularly since installing it a couple of weeks ago, and I have already wanted to open a new notebook from the console multiple times, so I thought others might find this useful too. 

Open to feedback and happy to change anything you like!

### CODE REVIEW

File: nbopen.py [hunk 1]
```diff
@@ -2,14 +2,15 @@
 
 import argparse
 import os.path
-import sys
```
Comment (senderle) [inline nbopen.py]: I removed this line since pyflakes indicated the `sys` module wasn't in use.

File: nbopen.py [hunk 2]
```diff
@@ -20,7 +21,7 @@ def find_best_server(filename, profile='default'):
     if profile != 'default':
         warnings.warn("Jupyter doesn't have profiles")
         kwargs['profile'] = profile
-    servers = [si for si in notebookapp.list_running_servers(**kwargs) \
```
Comment (senderle) [inline nbopen.py]: Another minor style thing -- apologies! I forgot that I had made these little stylistic changes.

File: setup.py [hunk 3]
```diff
@@ -55,8 +55,11 @@ def main(argv=None):
                   "License :: OSI Approved :: BSD License",
                   "Programming Language :: Python :: 2",
                   "Programming Language :: Python :: 3",
-                 ],
-             **extra_options)
+              ],
+              install_requires=[
+                  "nbformat"
+              ],
```
Comment (senderle) [inline setup.py]: Not sure about this requirement... seemed right though. ?

**Comment by takluyver** [comment]
Sorry that it's taken me over a year to look at this. Now that I have looked at it, I think it looks good. I'm going to merge it manually to resolve the conflict, and make a couple of minor cleanups.

**Pull request merged**